server/index.js

```diff
@@ -8,7 +8,7 @@ const REFRESH_DB = process.env.FORCE_REFRESH === 'true' || false;
 
 const startServer = (app, connection) => {
   return connection
-    .sync({force: REFRESH_DB })
+    .sync({force: true })
```

Comment:
Since we can create a .env file maybe we could leave force as an option?

Reply:
Sure! I only added changed to force true to view db changes after I added associations.. wasn't working until I changed to true.